### PR TITLE
adding back in data collected missed info

### PR DESCRIPTION
**Pull request by apigirl**
### What does this PR do?

Adds in info that we missed when we reorged the docs.

https://docs-staging.datadoghq.com/kaylyn/rum-missed-info/real_user_monitoring/browser/data_collected/

### CODE REVIEW

**Comment by apigirl** [comment]
@hdelaby - okay, I think I found it and added it!

**Comment by hdelaby** [comment]
Still some content is missing from the user action page, all the content before the attributes are listed.

File: content/en/real_user_monitoring/browser/data_collected.md [hunk 1]
```diff
@@ -38,25 +38,43 @@ A page view represents a user visiting a page from your website. During that pag
...
+
+Datadog provides a unique performance metric, `loading_time`, which calculates the time needed for a page to load. This metric works both for `initial_load` and `route_change` navigations.
+
+### How is loading time calculated?
+To account for modern web applications, loading time watches for network requests and DOM mutations.
+
+* **Initial Load**: Loading Time is equal to *whichever is longer*:
+
+    - The difference between `navigationStart` and `loadEventEnd`.
+    - Or the difference between `navigationStart` and the first time the page has no activity for more than 100ms (activity being defined as ongoing network requests, or DOM mutations are currently occurring).
```
Comment (kayayarai) [inline content/en/real_user_monitoring/browser/data_collected.md:53]:
```suggestion
    - Or the difference between `navigationStart` and the first time the page has no activity for more than 100ms (activity being defined as ongoing network requests or a DOM mutation).
```

File: content/en/real_user_monitoring/browser/data_collected.md [hunk 2]
```diff
@@ -38,25 +38,43 @@ A page view represents a user visiting a page from your website. During that pag
...
+
+* **SPA Route Change**: Loading Time is equal to the difference between the user click and the first time the page has no activity for more than 100ms (activity being defined as ongoing network requests, or DOM mutations are currently occurring)
```
Comment (kayayarai) [inline content/en/real_user_monitoring/browser/data_collected.md:55]:
```suggestion
* **SPA Route Change**: Loading Time is equal to the difference between the user click and the first time the page has no activity for more than 100ms (activity being defined as ongoing network requests, or a DOM mutation).
```

File: content/en/real_user_monitoring/browser/data_collected.md [hunk 3]
```diff
@@ -38,25 +38,43 @@ A page view represents a user visiting a page from your website. During that pag
...
-| `view.measures.dom_complete`           | number (ns) | The page and all the subresources are ready. For the user, the loading spinner has stopped spinning. [More info from the MDN documentation][11].                                                                             |
-| `view.measures.load_event_end`         | number (ns) | Event fired when the page is fully loaded. Usually a trigger for additional application logic. [More info from the MDN documentation][12].                                                                                   |
+| `view.loading_time`                             | number (ns) | Time until the page is ready and no network request or DOM mutation is currently occurring. [More info from the data collected documentation][8]|
+| `view.measures.first_contentful_paint` | number (ns) | Time when the browser first rendered any text, image (including background images), non-white canvas, or SVG. For more information about browser rendering, see the [w3 definition][9].                                                                                            |
```
Comment (kayayarai) [inline content/en/real_user_monitoring/browser/data_collected.md:69]:
```suggestion
| `view.measures.first_contentful_paint` | number (ns) | Time when the browser first renders any text, image (including background images), non-white canvas, or SVG. For more information about browser rendering, see the [w3 definition][9].                                                                                            |
```

File: content/en/real_user_monitoring/browser/data_collected.md [hunk 4]
```diff
@@ -38,25 +38,43 @@ A page view represents a user visiting a page from your website. During that pag
...
+| `view.measures.dom_interactive`        | number (ns) | The moment when the parser finished its work on the main document. [More info from the MDN documentation][10]                                                                                                               |
```
Comment (kayayarai) [inline content/en/real_user_monitoring/browser/data_collected.md:70]:
```suggestion
| `view.measures.dom_interactive`        | number (ns) | The moment when the parser finishes its work on the main document. [More info from the MDN documentation][10]                                                                                                               |
```

File: content/en/real_user_monitoring/browser/data_collected.md [hunk 5]
```diff
@@ -38,25 +38,43 @@ A page view represents a user visiting a page from your website. During that pag
...
+| `view.measures.dom_content_loaded`     | number (ns) | Event fired when the initial HTML document has been completely loaded and parsed, without waiting for non-render blocking stylesheets, images, and subframes to finish loading. [More info from the MDN documentation][11]. |
```
Comment (kayayarai) [inline content/en/real_user_monitoring/browser/data_collected.md:71]:
```suggestion
| `view.measures.dom_content_loaded`     | number (ns) | Event fired when the initial HTML document is completely loaded and parsed, without waiting for non-render blocking stylesheets, images, and subframes to finish loading. [More info from the MDN documentation][11]. |
```

File: content/en/real_user_monitoring/browser/data_collected.md [hunk 6]
```diff
@@ -65,17 +83,18 @@ For Single Page Applications (SPAs), performance metrics will only be available
...
+[11]: https://developer.mozilla.org/en-US/docs/Web/API/Document/DOMContentLoaded_event
+[12]: https://developer.mozilla.org/en-US/docs/Web/API/Window/DOMContentLoaded_event
+[13]: https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event
 {{% /tab %}}
 {{% tab "Resource" %}}
 
 All of your website’s resources are collected by default: images, XHRs, [XMLHttpRequest][1], CSS files, JS assets, and font files.
 
-Detailed network timing data regarding the loading of an application’s resources are being collected with the [Performance Resource Timing API][2].
+Detailed network timing data regarding the loading of an application’s resources are being collected with the [Performance Resource Timing API][2]
```
Comment (kayayarai) [inline content/en/real_user_monitoring/browser/data_collected.md:97]:
```suggestion
Detailed network timing data for the loading of an application’s resources are collected with the [Performance Resource Timing API][2].
```
Or:
The Performance Resource Timing API][2] collects detailed network timing data for the loading of an application's resources.

File: content/en/real_user_monitoring/browser/data_collected.md [hunk 7]
```diff
@@ -123,21 +142,27 @@ Detailed network timing data regarding the loading of an application’s resourc
 {{% /tab %}}
 {{% tab "Error" %}}
 
+Front-end errors are automatically collected with Real User Monitoring (RUM). The error message and stack trace are included when available.
```
Comment (kayayarai) [inline content/en/real_user_monitoring/browser/data_collected.md:145]:
```suggestion
Front-end errors are collected with Real User Monitoring (RUM). The error message and stack trace are included when available.
```

File: content/en/real_user_monitoring/browser/data_collected.md [hunk 8]
```diff
@@ -123,21 +142,27 @@ Detailed network timing data regarding the loading of an application’s resourc
...
 
-All errors that appear in the user console are collected.
+## Error Origins
+Front-end errors are split in 3 different categories depending on their `error.origin`:
+
+- **network**: XHR or Fetch errors resulting from AJAX requests. Specific attributes to network errors can be found [in the documentation][1]
+- **source**: unhandled exceptions or unhandled promise rejections (source-code related)
+- **console**: console.error() API calls
```
Comment (kayayarai) [inline content/en/real_user_monitoring/browser/data_collected.md:152]:
```suggestion
- **console**: `console.error()` API calls.
```

File: content/en/real_user_monitoring/browser/data_collected.md [hunk 9]
```diff
@@ -123,21 +142,27 @@ Detailed network timing data regarding the loading of an application’s resourc
 {{% /tab %}}
 {{% tab "Error" %}}
 
+Front-end errors are automatically collected with Real User Monitoring (RUM). The error message and stack trace are included when available.
 
-All errors that appear in the user console are collected.
+## Error Origins
+Front-end errors are split in 3 different categories depending on their `error.origin`:
+
+- **network**: XHR or Fetch errors resulting from AJAX requests. Specific attributes to network errors can be found [in the documentation][1]
```
Comment (kayayarai) [inline content/en/real_user_monitoring/browser/data_collected.md:150]:
```suggestion
- **network**: XHR or Fetch errors resulting from AJAX requests. Specific attributes to network errors can be found [in the documentation][1].
```

File: content/en/real_user_monitoring/browser/data_collected.md [hunk 10]
```diff
@@ -123,21 +142,27 @@ Detailed network timing data regarding the loading of an application’s resourc
...
+- **source**: unhandled exceptions or unhandled promise rejections (source-code related)
```
Comment (kayayarai) [inline content/en/real_user_monitoring/browser/data_collected.md:151]:
```suggestion
- **source**: Unhandled exceptions or unhandled promise rejections (source-code related).
```

File: content/en/real_user_monitoring/browser/data_collected.md [hunk 11]
```diff
@@ -123,21 +142,27 @@ Detailed network timing data regarding the loading of an application’s resourc
...
+- **console**: console.error() API calls
 
 ## Facet Collected
 
 | Attribute       | Type   | Description                                                       |
 |-----------------|--------|-------------------------------------------------------------------|
+| `error.origin`  | string | Where the error originates from (i.e. console, network, etc.)          |
```
Comment (kayayarai) [inline content/en/real_user_monitoring/browser/data_collected.md:158]:
```suggestion
| `error.origin`  | string | Where the error originates from (for example, the console or network).     |
```

File: content/en/real_user_monitoring/browser/data_collected.md [hunk 12]
```diff
@@ -123,21 +142,27 @@ Detailed network timing data regarding the loading of an application’s resourc
...
 | `error.kind`    | string | The error type or kind (or code in some cases).                   |
 | `error.message` | string | A concise, human-readable, one-line message explaining the event. |
 | `error.stack`   | string | The stack trace or complementary information about the error.     |
-| `error.origin`  | string | Where the error originates from (i.e. console, network, etc.)          |
 
-### Network Error
+### Network Errors
 
-For network errors, the following facets are also collected:
+Network errors include information about the failing HTTP request. Hence, the following facets are also collected:
```
Comment (kayayarai) [inline content/en/real_user_monitoring/browser/data_collected.md:165]:
```suggestion
Network errors include information about failing HTTP requests. The following facets are also collected:
```

File: content/en/real_user_monitoring/browser/data_collected.md [hunk 13]
```diff
@@ -123,21 +142,27 @@ Detailed network timing data regarding the loading of an application’s resourc
...
 | Attribute       | Type   | Description                                                       |
 |-----------------|--------|-------------------------------------------------------------------|
+| `error.origin`  | string | Where the error originates from (i.e. console, network, etc.)          |
 | `error.kind`    | string | The error type or kind (or code in some cases).                   |
 | `error.message` | string | A concise, human-readable, one-line message explaining the event. |
 | `error.stack`   | string | The stack trace or complementary information about the error.     |
-| `error.origin`  | string | Where the error originates from (i.e. console, network, etc.)          |
 
-### Network Error
+### Network Errors
```
Comment (kayayarai) [inline content/en/real_user_monitoring/browser/data_collected.md:163]:
```suggestion
### Network errors
```

File: content/en/real_user_monitoring/browser/data_collected.md [hunk 14]
```diff
@@ -148,19 +173,54 @@ For network errors, the following facets are also collected:
 | `http.url_details.queryString` | object | The HTTP query string parts of the URL decomposed as query params key/value attributes. |
 | `http.url_details.scheme`      | string | The protocol name of the URL (HTTP or HTTPS)                                            |
 
+### Source Errors
```
Comment (kayayarai) [inline content/en/real_user_monitoring/browser/data_collected.md:176]:
```suggestion
### Source errors
```

File: content/en/real_user_monitoring/browser/data_collected.md [hunk 15]
```diff
@@ -148,19 +173,54 @@ For network errors, the following facets are also collected:
...
+
+Source Errors include code-level information about the error. More information about the different error types can be found in [the MDN documentation][2].
```
Comment (kayayarai) [inline content/en/real_user_monitoring/browser/data_collected.md:178]:
```suggestion
Source errors include code-level information about the error. More information about the different error types can be found in [the MDN documentation][2].
```

File: content/en/real_user_monitoring/browser/data_collected.md [hunk 16]
```diff
@@ -148,19 +173,54 @@ For network errors, the following facets are also collected:
...
+| `error.kind`    | string | The error type or kind (or code in some cases).                   |
+
+[1]: /real_user_monitoring/data_collected/error/#network-errors
+[2]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error
 {{% /tab %}}
 {{% tab "User Action" %}}
 
-A custom user actions is a custom event generated for a given user action. [Add one by instrumenting your code][1].
+## Automatic Collection of User Actions
+Real User Monitoring (RUM) SDKs automatically detect user interactions performed during a user journey. Set the `trackInteractions` [initialization parameter][5] to `true` to enable this feature.
```
Comment (kayayarai) [inline content/en/real_user_monitoring/browser/data_collected.md:190]:
```suggestion
Real User Monitoring (RUM) SDKs detect user interactions performed during a user journey. Set the `trackInteractions` [initialization parameter][5] to `true` to enable this feature.
```

File: content/en/real_user_monitoring/browser/data_collected.md [hunk 17]
```diff
@@ -148,19 +173,54 @@ For network errors, the following facets are also collected:
...
 
-## Facet Collected
+**Note**:  The trackInteractions initialization parameter enables the automatic collection of user clicks in your application. **Sensitive and private data** contained on your pages may be included to identify the elements interacted with.
```
Comment (kayayarai) [inline content/en/real_user_monitoring/browser/data_collected.md:192]:
```suggestion
**Note**:  The `trackInteractions` initialization parameter enables the collection of user clicks in your application. **Sensitive and private data** contained on your pages may be included to identify the elements interacted with.
```

File: content/en/real_user_monitoring/browser/data_collected.md [hunk 18]
```diff
@@ -148,19 +173,54 @@ For network errors, the following facets are also collected:
...
+
+Once an interaction is detected, all new RUM events are attached to the ongoing user action until it is considered as finished. The user action also benefits from its parent view attributes: browser information, geolocation data, [global context][1], etc.
```
Comment (kayayarai) [inline content/en/real_user_monitoring/browser/data_collected.md:194]:
```suggestion
Once an interaction is detected, all new RUM events are attached to the ongoing user action until it is considered finished. The user action also benefits from its parent view attributes such as browser information, geolocation data, [global context][1].
```

File: content/en/real_user_monitoring/browser/data_collected.md [hunk 19]
```diff
@@ -148,19 +173,54 @@ For network errors, the following facets are also collected:
...
+
+### How is the User Action duration calculated?
+Once an interaction is detected, the RUM SDK watches for network requests an DOM mutations. It is considered finished once the page has no activity for more than 100ms (activity being defined as ongoing network requests, or DOM mutations are currently occurring).
```
Comment (kayayarai) [inline content/en/real_user_monitoring/browser/data_collected.md:197]:
```suggestion
Once an interaction is detected, the RUM SDK watches for network requests an DOM mutations. It is considered finished once the page has no activity for more than 100ms (activity being defined as ongoing network requests or DOM mutations).
```

File: content/en/real_user_monitoring/browser/data_collected.md [hunk 20]
```diff
@@ -148,19 +173,54 @@ For network errors, the following facets are also collected:
...
+
+## Custom User Actions
+Custom User Actions are User Actions declared and sent manually via the [`addUserAction` API][2]. They are useful to send information relative to an event occuring during a user journey: a custom timing, customer cart information, etc.
```
Comment (kayayarai) [inline content/en/real_user_monitoring/browser/data_collected.md:200]:
```suggestion
Custom User Actions are User Actions declared and sent manually via the [`addUserAction` API][2]. They can send information relative to an event occurring during a user journey, for example, a custom timing or customer cart information.
```

**Review by kayayarai — CHANGES_REQUESTED** [review]

**Review by kayayarai — APPROVED** [review]

**Pull request merged**